packages/web/src/pages/settings-page/components/desktop/SettingsPage.tsx

```diff
@@ -269,20 +271,15 @@ class SettingsPage extends Component<SettingsPageProps, SettingsPageState> {
         <div className={styles.settings}>
           {this.renderThemeCard()}
           <SettingsCard
-            icon={<IconVerified />}
-            title={messages.verificationCardTitle}
-            description={messages.verificationCardDescription}
+            icon={<IconMessage />}
```

Comment:
Just wanted to call out that I noticed the icons and text seem misaligned in the figma, added a comment asking julian about it there.